travis/setup_conda_windows.sh

```diff
@@ -6,12 +6,14 @@ if [[ $DEBUG == True ]]; then
 fi
 
 if [[ -z "${MINICONDA_VERSION}" ]]; then
-    MINICONDA_VERSION=4.5.4
+    MINICONDA_VERSION=4.6.14
 fi
 
 echo "installing miniconda3"
 choco install miniconda3 --params="'/AddToPath:1'" --version="$MINICONDA_VERSION";
-export PATH="/c/tools/miniconda3/scripts:/c/tools/miniconda3/:$PATH";
+export PATH="/c/tools/miniconda3/bin:$PATH"
```

Comment:
```suggestion
export PATH="/c/tools/miniconda3/scripts:/c/tools/miniconda3/bin:$PATH"
```

Reply:
I don't think `bin` exists at all:

```++++ ls -1 /c/tools/miniconda3/bin
ls: cannot access '/c/tools/miniconda3/bin': No such file or directory```

Reply:
Ah, can you figure out where it is installing?

Reply:
`conda.exe` seems to be in `scripts` as you have it. Conda does not create a `bin` directory on Windows so you can remove the second path.

I did just scripts and it found conda but it still didn't like using `conda activate base` without doing anything extra. I'll try source the bash profile and see what happens.

```

<details>

```

++++ ls -1 /c/tools/miniconda3/scripts
2to3-script.py
2to3.exe
activate
activate.bat
chardetect-script.py
chardetect.exe
conda-env-script.py
conda-env.exe
conda-script.py
conda.exe
deactivate
easy_install-script.py
easy_install.exe
idle-script.py
idle.exe
menuinst-script.py
menuinst.exe
pip-script.py
pip.exe
pydoc-script.py
pydoc.exe
pywin32_postinstall.py
pywin32_testall.py
wheel-script.py
wheel.exe
++++ conda init bash
no change     C:\tools\miniconda3\Scripts\conda.exe
no change     C:\tools\miniconda3\Scripts\conda-env.exe
no change     C:\tools\miniconda3\Scripts\conda-script.py
no change     C:\tools\miniconda3\Scripts\conda-env-script.py
no change     C:\tools\miniconda3\condabin\conda.bat
no change     C:\tools\miniconda3\Library\bin\conda.bat
no change     C:\tools\miniconda3\condabin\_conda_activate.bat
no change     C:\tools\miniconda3\condabin\rename_tmp.bat
no change     C:\tools\miniconda3\condabin\conda_auto_activate.bat
no change     C:\tools\miniconda3\condabin\conda_hook.bat
no change     C:\tools\miniconda3\Scripts\activate.bat
no change     C:\tools\miniconda3\condabin\activate.bat
no change     C:\tools\miniconda3\condabin\deactivate.bat
modified      C:\tools\miniconda3\Scripts\activate
modified      C:\tools\miniconda3\Scripts\deactivate
modified      C:\tools\miniconda3\etc\profile.d\conda.sh
modified      C:\tools\miniconda3\etc\fish\conf.d\conda.fish
no change     C:\tools\miniconda3\shell\condabin\Conda.psm1
modified      C:\tools\miniconda3\shell\condabin\conda-hook.ps1
modified      C:\tools\miniconda3\Lib\site-packages\xonsh\conda.xsh
modified      C:\tools\miniconda3\etc\profile.d\conda.csh
modified      C:\Users\travis\.bash_profile
==> For changes to take effect, close and re-open your current shell. <==
setting up common dependencies
++++ conda activate base
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
If using 'conda activate' from a batch script, change your
invocation to 'CALL conda.bat activate'.
    $ conda init <SHELL_NAME>
Currently supported shells are:
  - bash
  - cmd.exe
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell
See 'conda init --help' for more information and options.
IMPORTANT: You may need to close and restart your shell after running 'conda init'.
```

</details>